### PR TITLE
Delete and readd new versions of stuff

### PR DESCRIPTION
**Pull request by isaacojq11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 Templates for https://platform.seal.run by [Seal](https://seal.run).
 
-**[Internal-only]** See [Slack](https://opram.slack.com/archives/C0509TH14DN/p1732025689004189) for instructions.
\ No newline at end of file
+**[Internal-only]** See [Slack](https://opram.slack.com/archives/C0509TH14DN/p1732025689004189) for instructions.
+
+Bump
```
Comment (btabram) [inline README.md:7]: ?

Comment (isaacojq11) [reply README.md:7]: woops sorry that was Boyd showing me how I can edit stuff and it directly updates!

Comment (btabram) [reply README.md:7]: Haha no worries, if you remove this change and .DS_Store (I think that's some macos metadata thing) then this looks good to me 👍

Comment (isaacojq11) [reply README.md:7]: Cool!!

File: sop-staff-training-and-competency-assessment/iso-13485-template-pack/manifest.json [hunk 2]
```diff
@@ -0,0 +1,1202 @@
+{
```
Comment (btabram) [inline sop-staff-training-and-competency-assessment/iso-13485-template-pack/manifest.json:1]: Oh wait, you've also added the iso pack to this folder too. Can you remove it from there (and the other .DS_store file)?

**Pull request closed without merging**